lib/Controller/CallController.php

```diff
@@ -131,9 +132,10 @@ public function getPeersForCall($token) {
 	 * @UseSession
 	 *
 	 * @param string $token
+	 * @param int $flags
 	 * @return DataResponse
 	 */
-	public function joinCall($token) {
+	public function joinCall($token, $flags) {
```

Comment:
So would a user reconnect when enabling/disabling the camera/audio? Or would this just not change at all?

Reply:
For now the flags are only evaluated once as result to the `getUserMedia` when joining the call. Using that, the UI on the receiver side could look differently for users that don't have / allow access to their camera/microphone.

Updating the flags when the audio/video is enabled/disabled was not part of the scope of this PR (I would rather do that in a separate PR).